www/source/partials/docs/_dev-pkgs-composite-pkg.html.md.erb

```diff
@@ -0,0 +1,123 @@
+
+If you **do** change the identifier, (e.g. from `origin/composite-example` to `origin/composite-example/2.0`), then potentially different versions of a composite could have very different member services. For example, the latest version of `origin/composite` could reference different services compared to `origin/composite/2.0`. Some will be in common, some will be removed, and some will be added. In the case where services are removed, they will be unloaded when loading a different version of the composite. And any old configurations of services present in both the old and new composites are discarded completely in favor of the new values.
+
+If you do not always use the latest version of the composite by specifying `origin/package` as the identifier, you might not have a consistent set of services loaded, since Habitat runs the latest version of a service that will
+satisfy the identifier. 
+
+> **Note** This is not an issue in composites that contain services described by fully-qualified identifiers, since those exact versions would always
+be run. 
+
+
+### Additional Binding Support
+In addition to binding between services in a composite package, you can also bind to services outside of the composite package through the `--bind` option as you would between two standalone services. When loading composites, you specify a three-part binding with the following format:
+
+~~~
+$ hab svc load yourorigin/composite-example --bind <composite-service>:<bind_name>:<service_group>
+~~~
+
+This is effectively a "normal" binding, prefixed by the service in the composite it applies to. For this type of binding, composites can be thought of as one unified service (instead of as the container for multiple services that it is), so you can think of these 3-part binds as the binds of that one service.
+
+## Limitations
+While you can install, start, stop, load, and unload composite packages, they have the following limitations:
+
+* The individual services are still available and addressable just as though you'd started them manually. 
+* Composite packages are not directly addressable as though they were services themselves. 
+* They cannot do anything like export selected values from their constituent services as "composite-level exports", or map "composite-level binds" onto constituent service binds. 
+* A composite package as a whole does not update itself like an individual service can; however, the constituent services of a composite can continue to update independently of each other.
+* At this time, composite package support is only available for Linux services.
+* Composites will not function properly on any Supervisor running with an organization set (e.g. specifying `--org` when running `hab start`).
+
+The name of the composite service is displayed when running `hab svc status`. Also, services record which composite (if any) they are a member
```

Comment:
As currently structured, it looks like this `hab svc status` discussion is about a limitation of the current implementation; it should just be another feature.